src/main/resources/oss/fosslight/repository/PartnerMapper.xml

```diff
@@ -225,6 +228,9 @@
 		<if test="!@oss.fosslight.util.StringUtil@isEmpty(modifier)">
 		   AND A.MODIFIER LIKE CONCAT('%',#{modifier},'%')
 		</if>
+		<if test="!@oss.fosslight.util.StringUtil@isEmpty(description)">
+			AND A.DESCRIPTION LIKE CONCAT('%',#{description},'%')
+		</if>

```

Comment:
It looks better to modify it with the same indent 
```suggestion
		<if test="!@oss.fosslight.util.StringUtil@isEmpty(modifier)">
		   AND A.MODIFIER LIKE CONCAT('%',#{modifier},'%')
		</if>
		<if test="!@oss.fosslight.util.StringUtil@isEmpty(description)">
		   AND A.DESCRIPTION LIKE CONCAT('%',#{description},'%')
		</if>

```

Reply:
Thanks for your review! I'll change it. 
Can I see any code convention files related to this project?

Reply:
Someone else will be able to tell you the answer to your question.

Reply:
@cookienc 
The code convention has not set yet.
I will let you know when the code convention is set. 
Thanks. :)